frontend/mock-api/forms/export-form.json

```diff
@@ -80,6 +80,18 @@
                 ],
                 "validations": ["NOT_EMPTY"]
               },
+              {
+                "type": "HEADLINE",
+                "label": {
+                  "de": "Zeit davor und danach"
+                }
+              },
+              {
+                "type": "DESCRIPTION",
```

Comment:
@MaxVonKnobloch FYI: that's the way headline & description can be inserted from now on